sda/conode.go

```diff
@@ -101,3 +97,26 @@ func (c *Conode) Address() network.Address {
 func (c *Conode) GetService(name string) Service {
 	return c.serviceManager.Service(name)
 }
+
+// ProtocolRegisterName will sign up a new protocol to this Conode.
+// It returns the ID of the protocol.
+func (c *Conode) ProtocolRegisterName(name string, protocol NewProtocol) ProtocolID {
```

Comment:
I'm a bit hesitant to have two `ProtocolRegisterName` in sda, one in `sda.Conode` and one a the top lvl of `sda`. It seems they do the same thing while really the former has more power than the latter since it can overwrite the already registered `NewProtocol`. What do you think about `OverwriteNewProtocol` ? Whatever the name is, I'm not comfortable as using the same name.